span-processing-config-service-impl/src/main/java/org/hypertrace/span/processing/config/service/validation/SpanProcessingConfigRequestValidator.java

```diff
@@ -145,12 +144,8 @@ private void validateConfig(ApiNamingRuleConfig ruleConfig) {
         validateRegex(segmentMatchingBasedConfig.getRegexesList());
         break;
       case API_SPEC_BASED_CONFIG:
+        // TODO: Add validations for specId list after migration of upstream services
         ApiSpecBasedConfig apiSpecBasedConfig = ruleConfig.getApiSpecBasedConfig();
-        if (Strings.isNullOrEmpty(apiSpecBasedConfig.getApiSpecId())) {
```

Comment:
shouldn't we make it either spec id and spec ids list should be there otherwise its an invalid request.

Reply:
Right. Updated.